1_3.js

```diff
@@ -0,0 +1,21 @@
+let arr = ['掃除', '買い物', '散歩'];
+
+let print = (arr) => {
```

Comment:
関数を定義するなら `let` よりも `const` の方がより適切です。

```suggestion
const print = (arr) => {
```